.github/lint-scripts.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+#
+# lint bash scripts
+#
+
+# Copyright (c) 2019 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+
+set -o errexit
+
+CONFIG_DIR=".ci"
```

Comment:
should be `CONFIG_DIR=".github"` now as the scripts were moved

Reply:
ah thanks. missed that one :)